packages/markdown-it-myst/src/directives.ts

```diff
@@ -12,10 +12,10 @@ const COLON_OPTION_REGEX = /^:(?<option>[^:\s]+?):(\s*(?<value>.*)){0,1}\s*$/;
 function replaceFences(state: StateCore): boolean {
   for (const token of state.tokens) {
     if (token.type === 'fence' || token.type === 'colon_fence') {
-      const match = token.info.match(/^\{([^\s}]+)\}\s*(.*)$/);
+      const match = token.info.match(/^\s*\{([^}]+)\}\s*(.*)$/);
```

Comment:
This allows spaces _inside_ the directive name too. Probably don't want that?

Maybe `^\s*\{\s*([^}\s]+\s*)\}\s*(.*)$` instead...?

Reply:
Nice. Yeah, good point. I will update -- If we go with something more like pandoc in the future, this whole area is going to need an overhaul. But that is for later!

Reply:
Done!